### ISSUE TITLE
Feature Request: Wikilinks support

### ISSUE THREAD

**Issue by aboy021** [SublimeText-Markdown/MarkdownEditing#596]
Many markdown tools such as [Obsidian](https://obsidian.md/) support links using the wikilinks format.
For example, the wikilink 
`[[I am a link]]`
Corresponds to 
`[I am a link](I%20am%20a%20link.md)`
Unlike regular markdown links, wikilinks support unescaped whitespace.
 
It would be great if these links could be supported, possibly as a configuration option for GitHub Flavoured Markdown.

**Comment by reagle** [comment]
BTW: note that wikilink is a non-standard mess. [Pandoc will likely be able to support it soon](https://github.com/jgm/pandoc/issues/2923), but will require two extensions to cover `[[label|link]]` and ``[[link|label]]``.

**Comment by arcticpinecone** [comment]
> BTW: note that wikilink is a non-standard mess. [Pandoc will likely be able to support it soon](https://github.com/jgm/pandoc/issues/2923), but will require two extensions to cover `[[label|link]]` and `[[link|label]]`.

Is this possible now then? I would ❤️ this!!

**Comment by reagle** [comment]
I've not used it but the documentation [says so](https://pandoc.org/MANUAL.html#extension-wikilinks_title_after_pipe); this issue can be closed.

> ### Extension: wikilinks_title_after_pipe±
> Pandoc supports multiple Markdown wikilink syntaxes, regardless of whether the title is before or after the pipe.
>
> Using [--from=markdown+wikilinks_title_after_pipe](https://pandoc.org/MANUAL.html#option--from) results in
> 
> [[URL|title]]
>
> while using [--from=markdown+wikilinks_title_before_pipe](https://pandoc.org/MANUAL.html#option--from) results in
>
> [[title|URL]]

**Comment by naheller** [comment]
Hi there, I thought I'd ask here instead of opening a separate issue: does MarkdownEditing support auto-completion during link creation? For example, a dropdown of available files appearing after the user has entered `[[`.

**Comment by deathaxe** [comment]
No, it does not auto-complete links at the moment. Wikilink support is limited to opening a link (via F12 by default) and list "back links" via command palette.
